Examples/CustomMachine/jax_wrapper.py

```diff
@@ -0,0 +1,168 @@
+# Copyright 2019 The Simons Foundation, Inc. - All Rights Reserved.
```

Comment:
it would be nice to be able to load this as a part of the netket module... 
Something like `netket.machine.Jax()`